com.ibm.streamsx.topology/opt/python/packages/streamsx/rest_primitives.py

```diff
@@ -404,49 +441,6 @@ class Job(_ResourceElement):
         >>> print (jobs[0].health)
         healthy
     """
-    def get_application_logs(self, path=None, prefix=None):
```

Comment:
The `tester.py` module has a reference to `get_application_logs`. When a test fails, it would retrieve the logs:
https://github.com/ejpring/streamsx.topology/blob/814bb4f6e470d9f2fbc5e649c084bc012df7b825/com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/tester.py#L539

Could you update the reference to the new method name `store_logs`?

Reply:
I'm confused why are we removing `get_application_logs` ?